src/SprykerEco/Yves/Heidelpay/Theme/default/components/molecules/direct-debit-new-registration/direct-debit-new-registration.twig

```diff
@@ -0,0 +1,32 @@
+{% extends model('component') %}
+
+{% define config = {
+    name: 'direct-debit-new-registration',
+    tag: 'direct-debit-new-registration',
+} %}
+
+{% define data = {
+    paymentOption: data.paymentOption,
+    paymentMethod: data.paymentMethod,
```

Comment:
Please add `required` or type of optional value for example `''`, `{}`, etc...

Reply:
fixed